src/test/mir-opt/issue-38669.rs

```diff
@@ -25,27 +25,29 @@ fn main() {
 //     bb0: {
 //         StorageLive(_1);
 //         _1 = const false;
-//         goto -> bb1;
+//         goto -> bb2;
```

Comment:
This might also be a good spot for a mir-opt test targeting *exactly* this change. It'd be nice to have `fn foo() { loop { } }` and then dump the "build" MIR with a comment stating that we are checking that we emit `FalseEdges` as needed.